test/Nancy.Tests/Unit/DynamicDictionaryValueFixture.cs

```diff
@@ -987,6 +987,38 @@ public void Should_return_default_value_if_implicit_convert_fails_on_datetime()
         }
 
         [Fact]
+        public void Should_adjust_to_universal_time_when_globalizationconfiguration_datetimestyles_requires_it()
+        {
+            // Given
+            var expected = DateTime.Parse("2016-05-24T08:41:37.1661565");
+
+            var config = new GlobalizationConfiguration(new [] {"en-US"}, dateTimeStyles: DateTimeStyles.AdjustToUniversal);
+            var value = new DynamicDictionaryValue("2016-05-24T10:41:37.1661565+02:00", config);

```

Comment:
Not sure I understand what the test is trying to do, but a) I would set the expected value by calling the `DateTime` constructor and specifying the kind explicitly and b) if you're assuming a particular time zone, that's pretty worrying. However, if the time zone *is* set explicitly somewhere, then it's reasonable to assume that the rules for May 24th 2016 won't be changed in the future.